sdk/keyvault/azure-security-keyvault-secrets/pom.xml

```diff
@@ -47,6 +47,7 @@
     <!-- Configures the Java 9+ run to perform the required module exports, opens, and reads that are necessary for testing but shouldn't be part of the module-info. -->
     <javaModulesSurefireArgLine>
       --add-exports com.azure.core/com.azure.core.implementation.http=ALL-UNNAMED
+      --add-exports com.azure.core/com.azure.core.implementation.util=ALL-UNNAMED
```

Comment:
I take it you needed this for running local live tests?

Reply:
for running unit tests in JDK 17 locally, yeah.
This config doesn't impact users, mainly for local tests running only.

Reply:
Just as a note, we also needed this for Java 11, likely for anything using Java 9+? We're considering adding this to the parent POM. /cc @srnagar